### ISSUE TITLE
Center calendar and make it fit its content

### ISSUE THREAD

**Issue by FabianSefranek** [afonsocrg/mementoMori#2]
Add this to css:
```
body {
        display: flex;
        flex-direction: column;
        align-items: center;
}
```
and change in #calendar:
`width: fit-content;`

**Comment by afonsocrg** [comment]
This did not help aligning the calendar.

Instead, the css below helped
```
#calendar {
    width: 80%;
    margin: 0 auto;

    ...
}
```

**Comment by afonsocrg** [comment]
Fixed in commit ad386e36f7a6e3d34f571097e1fb6738b54ae181
